api/nakadi-event-bus-api.yaml

```diff
@@ -920,6 +920,12 @@ definitions:
               from the header into the metadata by Nakadi to avoid clients having to explicitly copy this.
             type: string
             example: 'JAh6xH4OQhCJ9PutIV_RYw'
+          partition:
```

Comment:
I have some further editorial suggestions about the api that I would edit in your branch directly if you don't mind.